internal/page/data.go

```diff
@@ -124,6 +125,10 @@ type Lpa struct {
 	WantToApplyForLpa bool
 	// Confirmation that the applicant wants to sign the LPA
 	WantToSignLpa bool
+	// CheckedAt is when the donor checked their LPA
+	CheckedAt time.Time
```

Comment:
I initially changed this because I was going to use it for the compare, but it would have ended up more complicated. I can put it back to a `bool` but think this is better anyway?

Reply:
Fine with me - leaves things open for future requests on when things have happened.